bitnami/aspnet-core/templates/deployment.yaml

```
@@ -133,10 +133,9 @@ spec:
             - name: BITNAMI_DEBUG
               value: {{ .Values.image.debug | quote }}
             {{- end }}
-            {{- range $key, $value := .Values.extraEnvVars }}
-            - name: {{ $key }}
-              value: "{{ $value }}"
-            {{- end }}
+            {{ - if .Values.extraEnvVars }}
+            {{ - include "common.tplvalues.render" (dict "value" .Values.extraEnvVars "context" $) | nindent 12 }}
+            {{ - end }}
```

Output of `helm template`:
```console
$ helm template .
Error: parse error at (aspnet-core/templates/deployment.yaml:136): "-"
```

```suggestion
            {{- if .Values.extraEnvVars }}
            {{- include "common.tplvalues.render" (dict "value" .Values.extraEnvVars "context" $) | nindent 12 }}
            {{- end }}
```

After the fix:
```
...
          env:
            - name: ASPNETCORE_URLS
              value: "http://+:8080"
            - name: FOO
              value: BAR
...
```

---

applied on [ca20290](https://github.com/bitnami/charts/pull/10857/commits/ca202902cb500eb19f22488b2cf0d92285a26142)! Thank you!